app/controllers/organizations_controller.rb

```diff
@@ -49,6 +49,14 @@ def set_current
     redirect_to root_path
   end
 
+  def select_organization
+    @members = Member.where(user_id: current_user.id)
+    @organizations = Array.new
+    @members.each do |member|
+      @organizations << Organization.find(member.organization_id)
+    end
```

Comment:
I think we can replace this entire code block by `@organizations = current_user.organizations`